### ISSUE TITLE
Error after upgrading to Ubuntu 18.10 (gnome-shell 3.30)

### ISSUE THREAD

**Issue by juergh** [home-sweet-gnome/dash-to-panel#499]
It seems the DisplayWrapper in utils.js is not working correctly with gnome-shell 3.30:
TypeError: global.screen is undefined

The extension loads fine after removing all instance of global.screen from DisplayWrapper.

**Comment by charlesg99** [comment]
Hello juergh, could you please add the error you're seeing in the logs? I'm on 3.30.1 (arch) and I've also tested specifically on ubuntu 18.10 and I see no such error. Also, what happens if you run `global.screen || global.display` in the looking glass (alt-f2 -> lg)? Thanks!

**Comment by juergh** [comment]
Hmmm... I realized that there is an Ubuntu package that provides the dash-to-panel extension (gnome-shell-extension-dash-to-panel). After installing that all worked well. I then uninstalled it again and installed the extension again manually from extensions.gnome.org, that worked as well. I'm not sure what I did different this time around but I can't reproduce the error any longer, so I'm closing the ticket. Sorry for the noise.
